lib/factory_bot/internal.rb

```diff
@@ -57,6 +75,30 @@ def register_factory(factory)
       def factory_by_name(name)
         factories.find(name)
       end
+
+      def register_strategy(strategy_name, strategy_class)
+        strategies.register(strategy_name, strategy_class)
+        StrategySyntaxMethodRegistrar.new(strategy_name).define_strategy_methods
+      end
+
+      def strategy_by_name(name)
+        strategies.find(name)
+      end
+
+      def register_default_strategies
+        DEFAULT_STRATEGIES.each do |strategy_name, strategy_class|
```

Comment:
Is it possible to do `DEFAULT_STRATEGIES.each(&method(:register_strategy))` like you did with the callbacks?